packages/component-dashboard/server/resolvers.js

```diff
@@ -18,6 +18,8 @@ const resolvers = {
       const userUuid = await models.User.getUuidForProfile(user)
       const manuscript = models.Manuscript.makeInitial({ createdBy: userUuid })
       manuscript.setDefaults()
+      await manuscript.save()
```

Comment:
This save needs to happen first as the id is generated on the `$beforeInsert`. We need the id before we can generate the url string

Reply:
Seems pedantic - but can we add this as a comment? Would be easy for someone to revisit this later and "remove the double-save".

Reply:
Yes, otherwise we can use uuid to create `id` here and pass to factory?